### ISSUE TITLE
Readme does not mention bokeh

### ISSUE THREAD

**Issue by philippjfr** [ioam/holoviews#781]
We've now had support for bokeh for almost a year and I just noticed our Readme doesn't even mention it. We should at least have some mention of it.

**Comment by jbednar** [comment]
Is https://github.com/ioam/holoviews/commit/ca01b8f90fd1cba2e1d9f3d8963d9a6f0d4906a0 sufficient?

**Comment by jlstevens** [comment]
Definitely an improvement. We probably need to think about what key information we really want to convey before we can decided whether it is sufficient. In general, I think the README could be trimmed down and improved...

**Comment by jlstevens** [comment]
I guess we could file a separate issue about the README in general. As it is, think @jbednar's commit is sufficient to address this issue for now.
